DatadogCore/Tests/Datadog/Tracing/TracingURLSessionHandlerTests.swift

```diff
@@ -178,11 +179,12 @@ class TracingURLSessionHandlerTests: XCTestCase {
         XCTAssertEqual(span.tags[OTTags.httpStatusCode], "404")
         XCTAssertEqual(span.tags[SpanTags.errorType], "HTTPURLResponse - 404")
         XCTAssertEqual(span.tags[SpanTags.errorMessage], "404 not found")
+        XCTAssertEqual(span.tags[SpanTags.kind], "client")
         XCTAssertEqual(
             span.tags[SpanTags.errorStack],
             "Error Domain=HTTPURLResponse Code=404 \"404 not found\" UserInfo={NSLocalizedDescription=404 not found}"
         )
-        XCTAssertEqual(span.tags.count, 8)
+        XCTAssertEqual(span.tags.count, 9)
```

Comment:
shouldn't this update the current tag that was `internal` before ? Why do we have more tags then before ? We were not sending the `kind:internal` before on these spans ?

Reply:
interesting enough we are not sending the kind as of now in open tracing world.

We do this in otel world.

Reply:
Very interesting ...